File: API/Entities/OrderProduct.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace API.Entities
{
    public class OrderProduct
    {
        public OrderProduct(int orderId, int productId, int quantity)
        {
            OrderId = orderId;
            ProductId = productId;
            Quantity = quantity;
        }

        public int OrderId { get; set; }

        public int ProductId { get; set; }

        public int Quantity { get; set; }
    }
}

File: API/ordermanager/src/app/_models/orderProduct.ts
export class OrderProduct {
  orderId: number;
  productId: number;
  quantity: number;

  constructor(_orderId: number, _productId: number, _quantity: number) {
    this.orderId = _orderId;
    this.productId = _productId;
    this.quantity = _quantity;
  }
}
